src/simulator/api.yaml

```diff
@@ -320,13 +320,8 @@ components:
       type: object
       required:
         - transferId
-        - quote

```

Comment:
Is there a test to prove that allowing incoming requests to this endpoint without these previously required properties does not cause errors? I do not see any unit tests to cover this change.

Reply:
You are right, there are no unit tests to test this change. I will implement as you suggested. Thanks.